benchmarks/redis/run.sh

```diff
@@ -125,15 +125,17 @@ done
 rm -rf results
 mkdir -p results
 
-docker run --name redis-docker -d --privileged $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM -p 7551:7551 --entrypoint="/root/entrypoint.sh" $CONTAINER "$MEMPOLICY"
+docker run --name redis-docker -d --privileged $MEMPOLICY $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM --network=host $CONTAINER 
+sleep 1
 
 start_time=$(date +%s%N)
-redis-benchmark -p 7551 -t GET,SET,LPUSH,LRANGE_100 --csv $DSIZE $THREADS > results/raw_results.txt
+docker run --name redis-benchmark --network=host --entrypoint=/usr/local/bin/redis-benchmark $CONTAINER  -p 7551 -t GET,SET,LPUSH,LRANGE_100 --csv $DSIZE $THREADS > results/raw_results.txt
 end_time=$(date +%s%N)
 time_taken=$(echo "scale=9; ($end_time - $start_time)/1000000000" | bc)
 
 docker container stop redis-docker
 docker container rm redis-docker
+docker container rm redis-benchmark
```

Comment:
Ideally, we would like to verify these commands succeed before proceeding and handle any errors appropriately.